File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-units_param.ts


import { ISearchUnits_Param } from './isearch-units_param';

export class SearchUnits_Param implements ISearchUnits_Param {
  
	public SearchString: string = "";
	public AgenciesOrUnits?: number;
	public CountryID?: number;
	public UnitMainAgencyID?: number;
	public PageSize?: number;
	public PageNumber?: number;
	public SortOrder: string = "";
	public SortDirection: string = "";
  
}

File: GTTS/INL.SearchService/Models/SearchUnits_Param.cs

namespace INL.SearchService.Models
{
    public class SearchUnits_Param : ISearchUnits_Param
    {
        public string SearchString { get; set; }
        /// <summary>
        /// 1: Agencies
        /// 2: Units
        /// 3: Both
        /// </summary>
        public int? AgenciesOrUnits { get; set; }
        public int? CountryID { get; set; }
        public long? UnitMainAgencyID { get; set; }
        public int? PageSize { get; set; }
        public int? PageNumber { get; set; }
        public string SortOrder { get; set; }
        public string SortDirection { get; set; }
    }
}
